
<!doctype html>
<html lang="en">
  <head>
  <script src="https://use.fontawesome.com/baff6f55f5.js"></script>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Teng LONG, Ph.D</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-light.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
      ga('create', 'UA-29643011-3', 'auto');
      ga('send', 'pageview');
    </script>

    <!-- New GA4 tracking code, see https://support.google.com/analytics/answer/10271001#analyticsjs-enable-basic --> 
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-GNJD50R0Z7"></script>
    <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());
    gtag('config', 'G-GNJD50R0Z7');
    </script>

    <!-- For all browsers -->
    <link rel="stylesheet" href="assets/css/academicons.min.css"/>
    <link rel="stylesheet" href="assets/css/academicons.css"/>
    
    <style>
      button.accordion {
      font:14px/1.5 Lato, "Helvetica Neue", Helvetica, Arial, sans-serif;
      cursor: pointer;
      padding: 0px;
      border: none;
      text-align: left;
      outline: none;
      font-size: 100%;
      transition: 0.3s;
      background-color: #f8f8f8;
      }
      button.accordion.active, button.accordion:hover {
      background-color: #f8f8f8;
      }
      button.accordion:after {
      content: " [+] ";
      font-size: 90%;
      color:#777;
      float: left;
      margin-left: 1px;
      }

      button.accordion.active:after {
      content: " [\2212] ";
      }
      div.panel {
      padding: 0 20px;
      margin-top: 5px;
      display: none;
      background-color: white;
      font-size: 100%;
      }
      div.panel.show {
      display: block !important;
      }
      .social-row {
        display: flex;
        flex-wrap: wrap;
        justify-content: space-between;
      }
    </style>
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Teng LONG, Ph.D</h1>
        <p><font size=2><i class="fa-solid fa-map-pin"></i> Hong Kong, China</font></p>
        <p>Postdoc Researcher in Virology<br><a href="https://www.hku.hk/">University of Hong Kong (HKU)</a></p>
        <p>Research Fellow <br><a href="https://www.c2i.hk/">Centre for Immunology &amp; Infection (C2I)</a></p><br><br>
    <h3><a href="https://longtjerry.github.io/">Home</a></h3>
    <h3><a href="https://longtjerry.github.io/index_CN">主頁</a></h3>
    <h3><a href="https://longtjerry.github.io/CV.html">About me</a></h3>
    <h3><a href="https://longtjerry.github.io/research.html">Research</a></h3> 
    <h3><a href="https://longtjerry.github.io/publications.html">Publications</a></h3>
    <h3><a href="https://longtjerry.github.io/collaborations.html">Collaborations</a></h3><br><br>
        
    <b>Social</b><br>
        <div class="social-row">
          <a href="mailto: jerrylongt302@gmail.com" class="author-social" target="_blank"><i class="fa-solid fa-envelope"></i> Email</a>
          <a href="https://scholar.google.com/citations?hl=zh-CN&user=HbpzOzoAAAAJ" target="_blank"><i class="ai ai-fw ai-google-scholar-square"></i> Google Scholar</a>
          <a href="https://orcid.org/0000-0002-6194-7885"><i class="fa-brands fa-orcid"></i> ORCID</a>
          <a href="https://www.researchgate.net/profile/Teng-Long-7" class="author-social" target="_blank"><i class="fa-brands fa-researchgate"></i> Research Gate</a>
          <a href="https://www.linkedin.com/in/tenglong302/" class="author-social" target="_blank"><i class="fa-brands fa-linkedin"></i> LinkedIn</a>
          <br>
        </div>
        <br>
    <p><b>Contact:</b><br>6/F, School of Public Health<br>Li Ka Shing Faculty of Medicine<br>21 Sassoon Road<br>Pok Fu Lam, Hong Kong Island</p><br>
    
      </header>
        
      <section>
    <!-- No R&Rs or WPs at the moment
    <h2><a id="recent-RRs-updated" class="anchor" href="#RRpapers" aria-hidden="true"><span class="octicon octicon-link"></span></a>Papers under Revision to Resubmit</h2>
    <p style="margin:0"> Coming soon. <br> <br> 

    <hr>

    <h2><a id="recent-papers-updated" class="anchor" href="#workingpapers" aria-hidden="true"><span class="octicon octicon-link"></span></a>Working Papers</h2>
    <p style="margin:0"> Coming soon. <br> <br> 

    <hr>                    
    -->
            
    <h1><b>Publications</b><font size=3> (<sup>1</sup>Co-first Author)</font></h1> 
    <br>
    <h3>2024</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://pubmed.ncbi.nlm.nih.gov/39095802/">SARS-CoV-2 variants divergently infect and damage cardiomyocytes in vitro and in vivo</a> <br>  Mok BW, Kwok M, Li HS, Ling L, Lai A, Yan B, Law CT, Yeung CH, Zhang AJ, Tam RC, Kukic A, Cremin CJ, Zhang Y, <b>Long T</b>, Kang Z, Luo R, Leung KT, Li AM, Lui G, Tsui SK, Chan JF, To KK, Chan PKS, Yan BP, Chen H, Poon EN. 2024. <i>Cell&amp;Bioscience</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p><b>BACKGROUND:</b> COVID-19 can cause cardiac complications and the latter are associated with poor prognosis and increased mortality. SARS-CoV-2 variants differ in their infectivity and pathogenicity, but how they affect cardiomyocytes (CMs) is unclear. <br><b>METHODS:</b> The effects of SARS-CoV-2 variants were investigated using human induced pluripotent stem cell-derived (hiPSC-) CMs in vitro and Golden Syrian hamsters in vivo. <br><b>RESULTS: </b>Different variants exhibited distinct tropism, mechanism of viral entry and pathology in the heart. Omicron BA.2 most efficiently infected and injured CMs in vitro and in vivo, and induced expression changes consistent with increased cardiac dysfunction, compared to other variants tested. Bioinformatics and upstream regulator analyses identified transcription factors and network predicted to control the unique transcriptome of Omicron BA.2 infected CMs. Increased infectivity of Omicron BA.2 is attributed to its ability to infect via endocytosis, independently of TMPRSS2, which is absent in CMs. <br><b>CONCLUSIONS:</b> In this study, we reveal previously unknown differences in how different SARS-CoV-2 variants affect CMs. Omicron BA.2, which is generally thought to cause mild disease, can damage CMs in vitro and in vivo. Our study highlights the need for further investigations to define the pathogenesis of cardiac complications arising from different SARS-CoV-2 variants. </p></div>     
    <br>     
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://pubmed.ncbi.nlm.nih.gov/38777146/">Identifications of novel host cell factors that interact with the receptor-binding domain of the SARS-CoV-2 spike protein</a> <br>  Tang X<sup>1</sup>, Liu Y<sup>1</sup>, Wang J<sup>1</sup>, <b>Long T<sup>1</sup></b>, Yee Mok BW, Huang Y, Peng Z, Jia Q, Liu C, So PK, Pui-Kam Tse S, Hei Ng C, Liu S, Sun F, Tang S, Yao ZP, Chen H, Guo Y. 2024. <i>Journal of Biological Chemistry</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> SARS-CoV-2 entry into host cells is facilitated by the interaction between the receptor-binding domain of its spike protein (CoV2-RBD) and host cell receptor, ACE2, promoting viral membrane fusion. The virus also uses endocytic pathways for entry, but the mediating host factors remain largely unknown. It is also unknown whether mutations in the RBD of SARS-CoV-2 variants promote interactions with additional host factors to promote viral entry. Here, we used the GST pull-down approach to identify novel surface-located host factors that bind to CoV2-RBD. One of these factors, SH3BP4, regulates internalization of CoV2-RBD in an ACE2-independent but integrin- and clathrin-dependent manner and mediates SARS-CoV-2 pseudovirus entry, suggesting that SH3BP4 promotes viral entry via the endocytic route. Many of the identified factors, including SH3BP4, ADAM9, and TMEM2, show stronger affinity to CoV2-RBD than to RBD of the less infective SARS-CoV, suggesting SARS-CoV-2-specific utilization. We also found factors preferentially binding to the RBD of the SARS-CoV-2 Delta variant, potentially enhancing its entry. These data identify the repertoire of host cell surface factors that function in the events leading to the entry of SARS-CoV-2. </p></div>
    <br> 
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.mdpi.com/1999-4915/16/6/981">Transmission Patterns of Co-Circulation of Omicron Sub-Lineages in Hong Kong SAR, China, a City with Rigorous Social Distancing Measures, in 2022</a> <br>  Chow N<sup>1</sup>, <b>Long T<sup>1</sup></b>, Lee L-K, Wong IT-F, Lee AW-T, Tam W-Y, Wong HF-T, Leung JS-L, Chow FW-N, Luk KS, Ho AY-M, Lam JY-W, Yau MC-Y, Que T-L, Yip K-T, Chow VC-Y, Wong RC-W, Mok BW-Y, Chen H-l, Siu GKH. 2024. <i>Viruses</i> [Brief Report]<br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> <b>Objective:</b> This study aimed to characterize the changing landscape of circulating SARS-CoV-2 lineages in the local community of Hong Kong throughout 2022. We examined how adjustments to quarantine arrangements influenced the transmission pattern of Omicron variants in a city with relatively rigorous social distancing measures at that time. <br><b>Methods:</b> In 2022, a total of 4684 local SARS-CoV-2 genomes were sequenced using the Oxford Nanopore GridION sequencer. SARS-CoV-2 consensus genomes were generated by MAFFT, and the maximum likelihood phylogeny of these genomes was determined using IQ-TREE. The dynamic changes in lineages were depicted in a time tree created by Nextstrain. Statistical analysis was conducted to assess the correlation between changes in the number of lineages and adjustments to quarantine arrangements. <br><b>Results:</b> By the end of 2022, a total of 83 SARS-CoV-2 lineages were identified in the community. The increase in the number of new lineages was significantly associated with the relaxation of quarantine arrangements (One-way ANOVA, F(5, 47) = 18.233, p < 0.001). Over time, Omicron BA.5 sub-lineages replaced BA.2.2 and became the predominant Omicron variants in Hong Kong. The influx of new lineages reshaped the dynamics of Omicron variants in the community without fluctuating the death rate and hospitalization rate (One-way ANOVA, F(5, 47) = 2.037, p = 0.091). Conclusion: This study revealed that even with an extended mandatory quarantine period for incoming travelers, it may not be feasible to completely prevent the introduction and subsequent community spread of highly contagious Omicron variants. Ongoing molecular surveillance of COVID-19 remains essential to monitor the emergence of new recombinant variants. </p></div>
    <br>
    <h3>2023</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.nature.com/articles/s41467-023-37697-1">An intranasal influenza virus-vectored vaccine prevents SARS-CoV-2 replication in respiratory tissues of mice and hamsters</a> <br>  Deng S, Liu Y, Tam RC, Chen P, Zhang AJ, Mok BW, <b>Long T</b>, Kukic A, Zhou R, Xu H, Song W, Chan JF, To KK, Chen Z, Yuen KY, Wang P, Chen H. 2023. <i>Nature Communications</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Current available vaccines for COVID-19 are effective in reducing severe diseases and deaths caused by SARS-CoV-2 infection but less optimal in preventing infection. Next-generation vaccines which are able to induce mucosal immunity in the upper respiratory to prevent or reduce infections caused by highly transmissible variants of SARS-CoV-2 are urgently needed. We have developed an intranasal vaccine candidate based on a live attenuated influenza virus (LAIV) with a deleted NS1 gene that encodes cell surface expression of the receptor-binding-domain (RBD) of the SARS-CoV-2 spike protein, designated DelNS1-RBD4N-DAF. Immune responses and protection against virus challenge following intranasal administration of DelNS1-RBD4N-DAF vaccines were analyzed in mice and compared with intramuscular injection of the BioNTech BNT162b2 mRNA vaccine in hamsters. DelNS1-RBD4N-DAF LAIVs induced high levels of neutralizing antibodies against various SARS-CoV-2 variants in mice and hamsters and stimulated robust T cell responses in mice. Notably, vaccination with DelNS1-RBD4N-DAF LAIVs, but not BNT162b2 mRNA, prevented replication of SARS-CoV-2 variants, including Delta and Omicron BA.2, in the respiratory tissues of animals. The DelNS1-RBD4N-DAF LAIV system warrants further evaluation in humans for the control of SARS-CoV-2 transmission and, more significantly, for creating dual function vaccines against both influenza and COVID-19 for use in annual vaccination strategies. </p></div>
    <br>
    <h3>2022</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1010444">Non-human primate papillomavirus E6-mediated p53 degradation reveals ancient evolutionary adaptation of carcinogenic phenotype to host niche</a> <br>  <b>Long T</b>, Burk, R.D., Chan, P.K.S., Chen, Z. 2022. <i>PloS Pathogens</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Non-human primates (NHPs) are infected with papillomaviruses (PVs) closely related to their human counterparts, but there are few studies on the carcinogenicity of NHP-PVs. Using an in vitro cell co-transfection assay, we systematically screened the biochemical activity of E6 proteins encoded by macaque PVs for their ability to bind and promote degradation of host p53 proteins. A host species barrier exists between HPV16 and MfPV3 with respect to E6-mediated p53 degradation that is reversed when p53 residue 129 is swapped between human and macaque hosts. Systematic investigation found that E6 proteins encoded by most macaque PV types in the high-risk species alpha12, but not other Alpha-PV clades or Beta-/Gamma-PV genera, can effectively promote monkey p53 degradation. Interestingly, two macaque PV types (MfPV10 and MmPV1) can simultaneously inhibit the expression of human and monkey p53 proteins, revealing complex cross-host interactions between PV oncogenes and host proteomes. Single point-mutant experiments revealed that E6 residue 47 directly interacts with p53 residue 129 for host-specific degradation. These findings suggest an ancient host niche adaptation toward a carcinogenic phenotype in high-risk primate PV ancestors. Following periods of primate host speciation, a loss-of-function mutation model could be responsible for the formation of a host species barrier to E6-mediated p53 degradation between HPVs and NHP-PVs. Our work lays a genetic and functional basis for PV carcinogenicity, which provides important insights into the origin and evolution of specific pathogens in host pathogenesis. </p></div>
    <br>
    <h3>2021</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.mdpi.com/2072-6694/13/11/2691">Current Updates on Cancer-Causing Types of Human Papillomaviruses (HPVs) in East, Southeast, and South Asia</a> <br>  Xia, C., Li, S., <b>Long T.</b>, Chen, Z., Chan, P.K.S., Boon, S.S. 2021. <i>Cancers</i> [Review] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Human papillomavirus (HPV) infection remains one of the most prominent cancer-causing DNA viruses, contributing to approximately 5% of human cancers. While association between HPV and cervical cancers has been well-established, evidence on the attribution of head and neck cancers (HNC) to HPV have been increasing in recent years. Among the cancer-causing HPV genotypes, HPV16 and 18 remain the major contributors to cancers across the globe. Nonetheless, the distribution of HPV genotypes in ethnically, geographically, and socio-economically diverse East, Southeast, and South Asia may differ from other parts of the world. In this review, we garner and provide updated insight into various aspects of HPV reported in recent years (2015-2021) in these regions. We included: (i) the HPV genotypes detected in normal cancers of the uterine cervix and head and neck, as well as the distribution of the HPV genotypes by geography and age groups; (ii) the laboratory diagnostic methods and treatment regimens used within these regions; and (iii) the oncogenic properties of HPV prototypes and their variants contributing to carcinogenesis. More importantly, we also unveil the similarities and discrepancies between these aspects, the areas lacking study, and the challenges faced in HPV studies. </p></div>
    <br>
    <h3>2020</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2020.00109/full">Phosphoprotein Gene of Wild-Type Rabies Virus Plays a Role in Limiting Viral Pathogenicity and Lowering the Enhancement of BBB Permeability</a> <br>  <b>Long T.</b>, Zhang B, Fan R, Wu Y, Mo M, Luo J, Chang Y, Tian Q, Mei M, Jiang H, Luo Y, Guo X. 2020. <i>Frontiers in Microbiology</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Enhancement of blood–brain barrier (BBB) permeability is necessary for clearing virus in the central nervous system (CNS). It has been reported that only laboratory-attenuated rabies virus (RABV) induces inflammatory response to lead BBB transient breakdown rather than wild-type (wt) strains. As a component of ribonucleoprotein (RNP), phosphoprotein (P) of RABV plays a key role in viral replication and pathogenicity. To our knowledge, the function of RABV P gene during RABV invasion was unclear so far. In order to determine the role of RABV P gene during RABV infection, we evaluated the BBB permeability in vivo after infection with wt RABV strain (GD-SH-01), a lab-attenuated RABV strain (HEP-Flury), and a chimeric RABV strain (rHEP-SH-P) whose P gene cloned from GD-SH-01 was expressed in the genomic backbone of HEP-Flury. We found that rHEP-SH-P caused less enhancement of BBB permeability and was less pathogenic to adult mice than GD-SH-01 and HEP-Flury. In an effort to investigate the mechanism, we found that the replication of rHEP-SH-P has been limited due to the suppressed P protein expression and induced less response to maintain BBB integrity. Our data indicated that the P gene of wt RABV was a potential determinant in hampering viral replication in vivo, which kept BBB integrity. These findings provided an important foundation for understanding the viral invasion and development of novel vaccine. </p></div>
    <br>
    <h3>2019</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2020.00109/full">Phenotypic Consequence of Rearranging the N Gene of RABV HEP-Flury</a> <br>  Mei M<sup>1</sup>, <b>Long T<sup>1</sup></b>, Zhang Q, Zhao J, Tian Q, Peng J, Luo J, Jiang H, Lin Y, Lin Z, Guo X. 2019. <i>Viruses</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Nucleoprotein (N) is a key element in rabies virus (RABV) replication. To further investigate the effect of N on RABV, we manipulated an infectious cDNA clone of the RABV HEP-Flury to rearrange the N gene from its wild-type position of 1 (N-P-M-G-L) to 2 (P-N-M-G-L), 3 (P-M-N-G-L), or 4 (P-M-G-N-L), using an approach that left the viral nucleotide sequence unaltered. Subsequently, viable viruses were recovered from each of the rearranged cDNA and examined for their gene expression levels, growth kinetics in cell culture, pathogenicity in suckling mice and protection in mice. The results showed that gene rearrangement decreased N mRNA transcription and vRNA replication. As a result, all viruses with rearranged genomes showed worse replication than that of rHEP-Flury in NA cells at a MOI of 0.01, but equivalent or slightly better replication levels at a MOI of 3. Consequently, the lethality in suckling mice infected with N4 was clearly attenuated compared with rHEP-Flury. However, the protection to mice was not enhanced. This study not only gives us insight into the understanding of the phenotype of RABV N gene rearrangement, but also helps with rabies vaccine candidate construction. </p></div>
    <br>    
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2019.02093/full">Non-human Primate Papillomaviruses Share Similar Evolutionary Histories and Niche Adaptation as the Human Counterparts</a> <br>  Chen Z<sup>1</sup>, <b>Long T<sup>1</sup></b>, Wong PY, Ho WCS, Burk RD, Chan PKS. 2019. <i>Frontiers in Microbiology</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Given high genetic diversity of papillomaviruses (PV) and complex scenario of virus-host interaction, the genetic basis underlying the mechanisms of HPV carcinogenicity is not well understood. In an effort to evaluate the origin and evolution of PV pathogenicity, we collected paired oral, perianal, and genital swabs from a wild macaque population. Of the 117 surveyed macaques, 88 (75.2%) were positive for PV DNA in one or more sites, mostly common from genital swabs, followed by oral and perianal sites. All putative macaque PV types phylogenetically clustered into the genera Alpha-, Beta-, and Gammapapillomavirus, with a strong phylogeny-tropism association as observed in HPVs. Using a Bayesian Markov Chain Monte Carlo framework, we demonstrated ancient intra-host divergence of primate PVs in which multiple ancestors had split and adapted to specific host ecosystems at least 41 million years ago, prior to the speciation events of primate host species. Following subsequent divergence and niche adaptation, distinct but phylogenetically related PV types were transmitted to similar host ecosystems by closely related host animals when host speciation occurred, which may explain in part the origin of carcinogenicity of HPV type 16 (HPV16) and Macaca fascicularis PV type 3 (MfPV3) that evolved from a most recent common ancestor containing the determinants for cervicovaginal colonization and cervical cancer. The findings identifying evolutionary and biological relatedness between human and non-human primate PVs lay a genetic foundation for research on parasite-host interactions and carcinogenic outcomes, which will prove useful in further study of viral pathogenesis and host specificity. <br><b>STUDY IMPORTANCE:</b> To better understand the origin and evolution of PV carcinogenicity associated with cervical cancer, we applied a combination of phylogenetic and bioinformatic analyses to investigate the genetic diversity of macaque papillomaviruses, and estimate divergence times of human and non-human primate PVs. The majority of both human and non-human primate PVs cluster into alpha-, beta-, and gamma-PVs, sharing similar evolutionary histories and biological properties to each other. The strong phylogeny-tropism association of primate PVs indicates an important role of niche adaptation and virus-host codivergence shaping the diversity of viral genomics, host specificity, immune exposure, and pathogenic property. Understanding the evolution of the family Papilloamviridae in general and the primate papillomaviruses in specific in relevant to virus-host interactions should provide important implications for viral pathogenesis and disease prevention. </p></div>
    <br>
    <h3>2018</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://journals.asm.org/doi/10.1128/mra.01414-18">Complete Genome Sequences of Six Novel Macaca mulatta Papillomavirus Types Isolated from Genital Sites of Rhesus Monkeys in Hong Kong SAR, China</a> <br>  <b>Long T.</b>, Wong PY, Ho WCS, Burk RD, Chan PKS, Chen Z. 2018. <i>Microbiology Resource Announcements</i> [Brief Report] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> The complete genomes of six Macaca mulatta papillomavirus types isolated from genital sites of rhesus monkeys were characterized, and less than 72% identity with the complete L1 genes of known papillomaviruses was found. Macaca mulatta papillomavirus type 2 (MmPV2), MmPV3, and MmPV6 cluster into the genus Alphapapillomavirus, and MmPV4, MmPV5, and MmPV7 cluster into the genus Gammapapillomavirus. </p></div>
    <br>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://link.springer.com/article/10.1007/s00705-018-3808-8">Recombinant rabies virus expressing interleukin-6 enhances the immune response in mouse brain</a> <br>  Luo J, Zhang B, Wu Y, Tian Q, Mo M, <b>Long T.</b>, Mei M, Fan R, Lyu Z, Jiang H, Wu F, Lin Y, Guo X. 2018. <i>Archives of Virology</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Rabies, which is caused by the rabies virus (RABV), is an ancient zoonosis that has a high mortality rate. Previous studies have indicated that recombinant RABV expressing canine interleukin-6 (rHEP-CaIL6), induced more virus-neutralizing antibodies than parental RABV in mice following intramuscular immunization. To investigate the immune response induced in the CNS by rHEP-CaIL6 after intranasal or intracranial administration in mice, the permeability of the blood-brain barrier (BBB), the infiltration of CD3 T cells, and innate immune response-related effector molecules in the CNS were examined. It was observed that infection of rHEP-CaIL6 led to enhanced BBB permeability following intranasal infection. More CD3 T cells infiltrated into the central nervous system (CNS) in mice infected with rHEP-CaIL6 than in those infected with the HEP-Flury strain. Furthermore, rHEP-CaIL6 induced an increased expression of innate immune response-related effector molecules, compared with the parental HEP-Flury strain, within the CNS. Taken together, these findings suggest that rHEP-CaIL6 induced stronger immune responses in mice brains, which is more beneficial for virus clearance. These results may also partly illustrate the role of IL6 in RABV infection. </p></div>
    <br>
    <h3>2017</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2017.01697/full">Phosphoprotein Gene Contributes to the Enhanced Apoptosis Induced by Wild-Type Rabies Virus GD-SH-01 In Vitro</a> <br>  Tian Q, Wang Y, Zhang Q, Luo J, Jiang H, Zhang B, Mei M, Wu F, Wu Y, Peng J, <b>Long T</b>, Luo Y, Guo X. 2017. <i>Frontiers in Microbiology</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Previous research demonstrated that the matrix protein (M) and glycoprotein (G) of attenuated rabies virus (RABV) strains are involved in the induction of host cell apoptosis. In this work, we show that wild-type (wt) RABV GD-SH-01 induces significantly greater apoptosis than the attenuated strain HEP-Flury. In order to identify the gene(s) accounting for this phenotype, five recombinant RABVs (rRABVs) were constructed by replacing each single gene of HEP-Flury with the corresponding gene of GD-SH-01. By using these rRABVs, we found that not only M and G, but also the phosphoprotein (P) plays an important role in inducing apoptosis. In order to figure out the different role of P gene in inducing apoptosis from the highly divergent background, another rRABV rGDSH-P, which carries the P gene of HEP-Flury in the background of the GD-SH-01 was generated. It was found that infection of NA cells with GD-SH-01 or the recombinant strain rHEP-shP, which carries P gene of GD-SH-01, induced significantly greater apoptosis than HEP-Flury or rGDSH-P in a caspase-dependent pathway that ultimately leads to the activation of the intrinsic apoptotic pathway, which is well characterized with the downregulation of bcl-2, the decrease of mitochondrial membrane potential, the release of mitochondrial cytochrome c, the activation of caspase-9 and caspase-3, and finally the cleavage of poly (ADP-ribose) polymerase. Our results imply that wt P from GD-SH-01 mediates this effect may partly by facilitating viral RNA synthesis but not by viral replication. In sum, we demonstrate a wt RABV strain GD-SH-01 to induce stronger apoptosis than an attenuated RABV HEP-Flury and propose that wt P from GD-SH-01 is involved in this process. </p></div>
    <br>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2017.00120/full">Phenotypic Consequences In vivo and In vitro of Rearranging the P Gene of RABV HEP-Flury</a> <br>  Mei M, <b>Long T</b>, Zhang Q, Zhao J, Tian Q, Peng J, Luo J, Wang Y, Lin Y, Guo X. 2017. <i>Frontiers in Microbiology</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Phosphoprotein (P) of the Rabies virus (RABV) is critically required for viral replication and pathogenicity. Here we manipulated infectious cDNA clones of the RABV HEP-Flury to translocate the P gene from its wild-type position 2 to 1, 3, or 4 in gene order, using an approach which left the viral nucleotide sequence unaltered. The recovered viruses were evaluated for the levels of gene expression, growth kinetics in cell culture, lethality in suckling mice and protection of mice. The results showed that viral replication was affected by the absolute value of N protein which was regulated by P protein. Viral lethality in suckling mice was consistent with the ratio of P mRNA in one complete transcription. The protection of mice induced by viruses was related to the antibody titer 5 weeks post-infection which might be regulated by G protein. However, the ability to induce cell apoptosis and viral spread were not only related to the viral replication but also to the ratio of related gene which affected by the gene position. These findings might not only improve the understanding of phenotype of RABV and P gene rearrangement, but also help rabies vaccine candidate construction. </p></div>
    <br>
    <h3>2016</h3>
    <p style="margin:0"> <a style="margin:0; font-size:100%; font-weight:bold" href="https://www.sciencedirect.com/science/article/abs/pii/S0264410X1630603X?via%3Dihub">Two potential recombinant rabies vaccines expressing canine parvovirus virion protein 2 induce immunogenicity to canine parvovirus and rabies virus</a> <br>  Luo J, Shi H, Tan Y, Niu X, <b>Long T</b>, Zhao J, Tian Q, Wang Y, Chen H, Guo X. 2016. <i>Vaccine</i> [Article] <br><button class="accordion">
    Abstract
    </button>
    <div class="panel" style="background-color: #F1F1F1; color: #666; padding: 10px;"><p> Both rabies virus (RABV) and canine parvovirus (CPV) cause lethal diseases in dogs. In this study, both high egg passage Flury (HEP-Flury) strains of RABV and recombinant RABV carrying double RABV glycoprotein (G) gene were used to express the CPV virion protein 2 (VP2) gene, and were designated rHEP-VP2 and, rHEP-dG-VP2 respectively. The two recombinant RABVs maintained optimal virus titration according to their viral growth kinetics assay compared with the parental strain HEP-Flury. Western blotting indicated that G protein and VP2 were expressed in vitro. The expression of VP2 in Crandell feline kidney cells post-infection by rHEP-VP2 and rHEP-dG-VP2 was confirmed by indirect immunofluorescence assay with antibody against VP2. Immunogenicity of recombinant rabies viruses was tested in Kunming mice. Both rHEP-VP2 and rHEP-dG-VP2 induced high levels of rabies antibody compared with HEP-Flury. Mice immunized with rHEP-VP2 and rHEP-dG-VP2 both had a high level of antibodies against VP2, which can protect against CPV infection. A challenge experiment indicated that more than 80% mice immunized with recombinant RABVs survived after infection of challenge virus standard 24 (CVS-24). Together, this study showed that recombinant RABVs expressing VP2 induced protective immune responses to RABV and CPV. Therefore, rHEP-VP2 and rHEP-dG-VP2 might be potential combined vaccines for RABV and CPV. </p></div>
    <br>
          
      </section>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    <script> 
    var acc = document.getElementsByClassName("accordion");
    var i;

    for (i = 0; i < acc.length; i++) {
        acc[i].onclick = function(){
            this.classList.toggle("active");
            this.parentNode.nextElementSibling.classList.toggle("show");
      }
    }
    </script>
  </body>
</html>
